site/public/js/calendar.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Gets the previous month of a given month
+ * @param month : int the current month (1 as January and 12 as December)
+ * @param year : int the current year
+ * @returns {number[]} : array<int> {previous_month, year_of_previous_month}
+ */
+// eslint-disable-next-line no-unused-vars
+function prevMonth(month, year) {
```

Comment:
you can implement prevMonth like so:
```js
//month is int 0=Jan, 11=Dec, 12=Jan, -1=Dec, ... 
prevMonth(month){
   let tmpDate = new Date();
   tmpDate.setMonth( month - 1 ); // date will handle looping for us
   return tmpDate.getMonth() //return month as int 0-11
}
```

Less math for us to worry about that way

Reply:
I'd prefer to let the month use 1-12 since the associative arrays storing the gradeable info in php are using 'Y-m-d' formatted strings (which use 1-12 as month), and also php date object is using 1-12 as month... Just to keep them consistent and intuitive

Reply:
ok, sounds good